quickstarts/Jumpstart - Quickstart/config.yml

```diff
@@ -0,0 +1,43 @@
+# Name of the quickstart (required)
+name: Jumpstart - Quickstart
+
+# Displayed in the UI (required)
+title: Jumpstart - Quickstart
+
+# Long-form description of the quickstart (required)
+description: |
+  The Jumpstart - Quickstart is a tool that has everything you need to help you get started building your own quickstart. It has some basic steps to follow to get started, some sample dashboards and queries, as well as a list of resources to help you get started!
+
+# Displayed in search results and recommendations. Summarizes a quickstarts functionality.
+summary: |
+  A Quickstart to help you get started building your own quickstart!
+
+# Support level: New Relic | Verified | Community (required)
+level: Community
+
+# Authors of the quickstart (required)
+authors:
+  - New Relic
+
+# Keywords for filtering / searching criteria in the UI
+keywords:
+  - Jumpstart
+  - Quickstart
+  - Start
+  - Started
+  - Intro
+
+# Reference to install plans located under /install directory
+# Allows us to construct reusable "install plans" and just use their ID in the quickstart config
+installPlans:
```

Comment:
we discussed using a custom 3rd party installer for this quickstart and not trying to leverage an existing one,  let's do`third-party/jumpstart`

https://github.com/newrelic/newrelic-quickstarts/tree/main/install/third-party